tests/acceptance/features/cliNotifications/sendNotifications.feature

```diff
@@ -0,0 +1,86 @@
+@cli @notification-app-required
+Feature: send notifications using the occ command
+
+  As an administrator
+  I want to be able to send notifications to the owncloud users
+  So that I can inform them about the necessary things
+
+  Scenario: administrator should be able to send a notification with subject and message to a user
+    Given these users have been created:
+      | username |
+      | user0    |
+    When the administrator sends following notifications using the occ command
+     | subject       | message                                          | user  |
+     | Quota updated | Congratulations your oC quota has been increased | user0 |
+    Then the command should have been successful
+    And user "user0" should have 1 notifications
+    And the last notification of user "user0" should match
+      | subject | Quota updated                                    |
+      | message | Congratulations your oC quota has been increased |
+
+  Scenario: administrator should be able to send a notification with a link
+    Given these users have been created:
+      | username |
+      | user0    |
+    When the administrator sends following notifications using the occ command
+     | subject       | message                                          | user  | link                                                                      |
+     | Quota updated | Congratulations your oC quota has been increased | user0 | https://owncloud.org/blog/about-activities-and-notifications-in-owncloud/ |
+    Then the command should have been successful
+    And user "user0" should have 1 notifications
+    And the last notification of user "user0" should match
+      | subject | Quota updated                                                             |
+      | message | Congratulations your oC quota has been increased                          |
+      | link    | https://owncloud.org/blog/about-activities-and-notifications-in-owncloud/ |
+
+  Scenario: administrator should be able to send a notification with only the subject to a user
+    Given these users have been created:
+      | username |
+      | user0    |
+    When the administrator sends following notifications using the occ command
+     | subject       | user  |
+     | Quota updated | user0 |
+    Then the command should have been successful
+    And user "user0" should have 1 notifications
+    And the last notification of user "user0" should match
+      | subject | Quota updated |
+
+  Scenario: administrator should be able to send a notification with subject and message to a group
+    Given these users have been created:
```

Comment:
May be we should check if all the members in a group are getting the notification or not. Try adding at least one more user to the group and check if both the users have notification or not.